src/explore-education-statistics-frontend/src/components/PrintThisPage.tsx

```diff
@@ -6,7 +6,17 @@ import {
 } from '@frontend/services/googleAnalyticsService';
 import styles from './PrintThisPage.module.scss';
 
-const PrintThisPage = ({ analytics, ...props }: AnalyticProps) => {
+interface Props {
+  noMargin?: boolean;
+  alignCentre?: boolean;
```

Comment:
A better prop here would just be a single `className`. These props are too specific to the visual styling and will probably have limited usefulness outside of the context of `PermalinkPage`.

Reply:
changed to single className prop